test/support/encoded_schema.json

```diff
@@ -1,10 +1,13 @@
 {
   "openapi": "3.0.0",
   "info": {
-    "contact": {},
+    "contact": {
+      "x-extension": "foo"
```

Comment:
I suggest replacing `x-extension: foo` with something more similar to an actual use case.
It could be something which shows this can even be be a map.

```yaml
x-custom-info:
  codeowners: team-rocker
```